### PR TITLE
Last minute Dutch string updates

### PR DESCRIPTION
**Pull request by ghost**
Weblate is indeed a disaster

### CODE REVIEW

File: config/locales/nl.yml [hunk 1]
```diff
@@ -11,7 +11,7 @@ nl:
     description_headline: Wat is %{domain}?
     domain_count_after: andere servers
     domain_count_before: Verbonden met
-    extended_description_html: |
+    extended_description_html:  |
```
Comment (ykzts) [inline config/locales/nl.yml]: remove whitespace.

File: config/locales/nl.yml [hunk 2]
```diff
@@ -24,7 +24,7 @@ nl:
       within_reach_body: Meerdere apps voor iOS, Android en andere platformen, met dank aan het ontwikkelaarsvriendelijke API-systeem, zorgen ervoor dat je overal op de hoogte blijft.
       within_reach_title: Altijd binnen bereik
     find_another_instance: Vind een andere server
-    generic_description: "%{domain} is een server in het Mastodon-netwerk"
+    generic_description: '%{domain} is een server in het Mastodon-netwerk'
```
Comment (ykzts) [inline config/locales/nl.yml]: `'` -> `"`

File: config/locales/nl.yml [hunk 3]
```diff
@@ -39,7 +39,7 @@ nl:
     followers: Volgers
     following: Volgend
     media: Media
-    moved_html: "%{name} is verhuisd naar %{new_profile_link}:"
+    moved_html: '%{name} is verhuisd naar %{new_profile_link}:'
```
Comment (ykzts) [inline config/locales/nl.yml]: `'` -> `"`

File: config/locales/nl.yml [hunk 4]
```diff
@@ -192,7 +194,7 @@ nl:
         create: Blokkade aanmaken
         hint: Een domeinblokkade voorkomt niet dat accountgegevens van dit domein aan de database worden toegevoegd, maar dat er met terugwerkende kracht en automatisch bepaalde moderatiemethoden op deze accounts worden toegepast.
         severity:
-          desc_html: "<strong>Negeren</strong> zorgt ervoor dat berichten van accounts van dit domein voor iedereen onzichtbaar zijn, behalve als een account wordt gevolgd. <strong>Opschorten</strong> zorgt ervoor dat alle berichten, media en profielgegevens van accounts van dit domein worden verwijderd. Gebruik <strong>Geen</strong> wanneer je alleen mediabestanden wilt weigeren."
+          desc_html: <strong>Negeren</strong> zorgt ervoor dat berichten van accounts van dit domein voor iedereen onzichtbaar zijn, behalve als een account wordt gevolgd. <strong>Opschorten</strong> zorgt ervoor dat alle berichten, media en profielgegevens van accounts van dit domein worden verwijderd. Gebruik <strong>Geen</strong> wanneer je alleen mediabestanden wilt weigeren.
```
Comment (ykzts) [inline config/locales/nl.yml]: Add `"`

File: config/locales/nl.yml [hunk 5]
```diff
@@ -207,7 +209,7 @@ nl:
       show:
         affected_accounts:
           one: Eén account in de database aangepast
-          other: "%{count} accounts in de database aangepast"
+          other: '%{count} accounts in de database aangepast'
```
Comment (ykzts) [inline config/locales/nl.yml]: `'` -> `"`

File: config/locales/nl.yml [hunk 6]
```diff
@@ -327,10 +329,10 @@ nl:
     title: Beheer
   admin_mailer:
     new_report:
-      body: "%{reporter} heeft %{target} gerapporteerd"
+      body: '%{reporter} heeft %{target} gerapporteerd'
```
Comment (ykzts) [inline config/locales/nl.yml]: `'` -> `"`

File: config/locales/nl.yml [hunk 7]
```diff
@@ -327,10 +329,10 @@ nl:
...
       subject: Nieuwe toots gerapporteerd op %{instance} (#%{id})
   application_mailer:
-    salutation: "%{name},"
+    salutation: '%{name},'
```
Comment (ykzts) [inline config/locales/nl.yml]: `'` -> `"`

File: config/locales/nl.yml [hunk 8]
```diff
@@ -370,18 +372,18 @@ nl:
...
-      over_x_years: "%{count}j"
-      x_days: "%{count}d"
-      x_minutes: "%{count}m"
-      x_months: "%{count}ma"
-      x_seconds: "%{count}s"
+      over_x_years: '%{count}j'
+      x_days: '%{count}d'
+      x_minutes: '%{count}m'
+      x_months: '%{count}ma'
+      x_seconds: '%{count}s'
```
Comment (ykzts) [inline config/locales/nl.yml]: `'` -> `"`

File: config/locales/nl.yml [hunk 9]
```diff
@@ -449,14 +451,14 @@ nl:
     generate: Genereren
     max_uses:
       one: 1 keer
-      other: "%{count} keer"
+      other: '%{count} keer'
```
Comment (ykzts) [inline config/locales/nl.yml]: `'` -> `"`

File: config/locales/nl.yml [hunk 10]
```diff
@@ -449,14 +451,14 @@ nl:
...
     max_uses_prompt: Onbeperkt
     prompt: Genereer en deel speciale links om mensen toegang tot deze Mastodon-server te geven
     table:
       expires_at: Verloopt op
       uses: Aantal keer te gebruiken
     title: Mensen uitnodigen
-  landing_strip_html: "<strong>%{name}</strong> is een gebruiker op %{link_to_root_path}. Je kunt deze volgen en ermee communiceren als je op Mastodon (of ergens anders in de fediverse) een account hebt."
+  landing_strip_html: <strong>%{name}</strong> is een gebruiker op %{link_to_root_path}. Je kunt deze volgen en ermee communiceren als je op Mastodon (of ergens anders in de fediverse) een account hebt.
```
Comment (ykzts) [inline config/locales/nl.yml]: Add double quotes.

**Review by ykzts — DISMISSED** [review]

**Comment by Gargron** [comment]
Be careful when messing with quotes, now they're not normalized:

```
The following data requires normalization:

config/locales/nl.yml

Run `i18n-tasks normalize` to fix
```

**Review by ghost — CHANGES_REQUESTED** [review]
_Please consider following the [documentation](https://github.com/tootsuite/documentation/blob/master/Contributing-to-Mastodon/Translating.md#appendix-b-command-tools) if you want to fix this yourself._
Otherwise please fix the following.

File: config/locales/nl.yml [hunk 11]
```diff
@@ -705,4 +708,4 @@ nl:
   users:
     invalid_email: E-mailadres is ongeldig
     invalid_otp_token: Ongeldige tweestaps-aanmeldcode
-    signed_in_as: 'Ingelogd als:'
+    signed_in_as: "Ingelogd als:"
```
Comment (SerCom-KC) [inline config/locales/nl.yml]: Please change it back to `'`

File: config/locales/nl.yml [hunk 12]
```diff
@@ -694,7 +697,7 @@ nl:
     enabled: Tweestapsverificatie is ingeschakeld
     enabled_success: Inschakelen tweestapsverificatie geslaagd
     generate_recovery_codes: Herstelcodes genereren
-    instructions_html: "<strong>Scan deze QR-code in Google Authenticator of een soortgelijke app op jouw mobiele telefoon</strong>. Van nu af aan genereert deze app aanmeldcodes die je bij het aanmelden moet invoeren."
+    instructions_html: <strong>Scan deze QR-code in Google Authenticator of een soortgelijke app op jouw mobiele telefoon</strong>. Van nu af aan genereert deze app aanmeldcodes die je bij het aanmelden moet invoeren.
```
Comment (SerCom-KC) [inline config/locales/nl.yml]: Please add `"` back

File: config/locales/nl.yml [hunk 13]
```diff
@@ -606,6 +608,7 @@ nl:
       public_long: Aan iedereen tonen, ook op openbare tijdlijnen
       unlisted: Minder openbaar
       unlisted_long: Aan iedereen tonen, maar niet op openbare tijdlijnen
+    title: '%{name}: "%{quote}"'
```
Comment (SerCom-KC) [inline config/locales/nl.yml]: Please move this line below [line 603](https://github.com/jeroenpraat/mastodon/blob/cb5527e73002c60fda85514052a0e2c2d9ed68cc/config/locales/nl.yml#L603)

File: config/locales/nl.yml [hunk 14]
```diff
@@ -535,7 +537,7 @@ nl:
     acct: Geef jouw account@domein.tld op waarvandaan je wilt volgen
     missing_resource: Kon vereiste doorverwijzings-URL voor jouw account niet vinden
     proceed: Ga door om te volgen
-    prompt: 'Jij gaat volgen:'
+    prompt: "Jij gaat volgen:"
```
Comment (SerCom-KC) [inline config/locales/nl.yml]: Please change it back to `'`

File: config/locales/nl.yml [hunk 15]
```diff
@@ -467,35 +469,35 @@ nl:
...
       body: "%{name} wil jou graag volgen"
-      subject: 'Volgen in afwachting: %{name}'
+      subject: "Volgen in afwachting: %{name}"
     mention:
-      body: 'Jij bent door %{name} vermeld in:'
+      body: "Jij bent door %{name} vermeld in:"
       subject: Jij bent vermeld door %{name}
     reblog:
-      body: 'Jouw toot werd door %{name} geboost:'
+      body: "Jouw toot werd door %{name} geboost:"
```
Comment (SerCom-KC) [inline config/locales/nl.yml]: Please change it back to `'`

File: config/locales/nl.yml [hunk 16]
```diff
@@ -467,35 +469,35 @@ nl:
...
     follow:
       body: "%{name} volgt jou nu!"
       subject: "%{name} volgt jou nu"
     follow_request:
       body: "%{name} wil jou graag volgen"
-      subject: 'Volgen in afwachting: %{name}'
+      subject: "Volgen in afwachting: %{name}"
     mention:
-      body: 'Jij bent door %{name} vermeld in:'
+      body: "Jij bent door %{name} vermeld in:"
```
Comment (SerCom-KC) [inline config/locales/nl.yml]: Please change it back to `'`

File: config/locales/nl.yml [hunk 17]
```diff
@@ -467,35 +469,35 @@ nl:
...
-      body: 'Jouw toot werd door %{name} als favoriet gemarkeerd:'
+      body: "Jouw toot werd door %{name} als favoriet gemarkeerd:"
       subject: "%{name} markeerde jouw toot als favoriet"
     follow:
       body: "%{name} volgt jou nu!"
       subject: "%{name} volgt jou nu"
     follow_request:
       body: "%{name} wil jou graag volgen"
-      subject: 'Volgen in afwachting: %{name}'
+      subject: "Volgen in afwachting: %{name}"
```
Comment (SerCom-KC) [inline config/locales/nl.yml]: Please change it back to `'`

File: config/locales/nl.yml [hunk 18]
```diff
@@ -467,35 +469,35 @@ nl:
...
         one: Jij hebt een nieuwe volger! Hoera!
         other: Jij hebt %{count} nieuwe volgers! Prachtig!
       subject:
-        one: "1 nieuwe melding sinds jouw laatste bezoek \U0001F418"
-        other: "%{count} nieuwe meldingen sinds jouw laatste bezoek \U0001F418"
+        one: 1 nieuwe melding sinds jouw laatste bezoek 🐘
+        other: "%{count} nieuwe meldingen sinds jouw laatste bezoek 🐘"
     favourite:
-      body: 'Jouw toot werd door %{name} als favoriet gemarkeerd:'
+      body: "Jouw toot werd door %{name} als favoriet gemarkeerd:"
```
Comment (SerCom-KC) [inline config/locales/nl.yml]: Please change it back to `'`

File: config/locales/nl.yml [hunk 19]
```diff
@@ -467,35 +469,35 @@ nl:
...
+      body: "Hier is een korte samenvatting van wat je hebt gemist op %{instance} sinds jouw laatste bezoek op %{since}:"
       mention: "%{name} vermeldde jou in:"
       new_followers_summary:
         one: Jij hebt een nieuwe volger! Hoera!
         other: Jij hebt %{count} nieuwe volgers! Prachtig!
       subject:
-        one: "1 nieuwe melding sinds jouw laatste bezoek \U0001F418"
-        other: "%{count} nieuwe meldingen sinds jouw laatste bezoek \U0001F418"
+        one: 1 nieuwe melding sinds jouw laatste bezoek 🐘
+        other: "%{count} nieuwe meldingen sinds jouw laatste bezoek 🐘"
```
Comment (SerCom-KC) [inline config/locales/nl.yml]: Please change `🐘` back to `\U0001F418`

File: config/locales/nl.yml [hunk 20]
```diff
@@ -467,35 +469,35 @@ nl:
...
-      body: 'Hier is een korte samenvatting van wat je hebt gemist op %{instance} sinds jouw laatste bezoek op %{since}:'
+      body: "Hier is een korte samenvatting van wat je hebt gemist op %{instance} sinds jouw laatste bezoek op %{since}:"
       mention: "%{name} vermeldde jou in:"
       new_followers_summary:
         one: Jij hebt een nieuwe volger! Hoera!
         other: Jij hebt %{count} nieuwe volgers! Prachtig!
       subject:
-        one: "1 nieuwe melding sinds jouw laatste bezoek \U0001F418"
-        other: "%{count} nieuwe meldingen sinds jouw laatste bezoek \U0001F418"
+        one: 1 nieuwe melding sinds jouw laatste bezoek 🐘
```
Comment (SerCom-KC) [inline config/locales/nl.yml]: Please change `🐘` back to `\U0001F418` and add `"` back

File: config/locales/nl.yml [hunk 21]
```diff
@@ -467,35 +469,35 @@ nl:
...
-    currently_redirecting: 'Jouw profiel wordt nu doorverwezen naar:'
+    currently_redirecting: "Jouw profiel wordt nu doorverwezen naar:"
     proceed: Opslaan
     updated_msg: Jouw accountmigratie-instelling is succesvol bijgewerkt!
   moderation:
     title: Moderatie
   notification_mailer:
     digest:
-      body: 'Hier is een korte samenvatting van wat je hebt gemist op %{instance} sinds jouw laatste bezoek op %{since}:'
+      body: "Hier is een korte samenvatting van wat je hebt gemist op %{instance} sinds jouw laatste bezoek op %{since}:"
```
Comment (SerCom-KC) [inline config/locales/nl.yml]: Please change it back to `'`

File: config/locales/nl.yml [hunk 22]
```diff
@@ -467,35 +469,35 @@ nl:
       too_many: Er kunnen niet meer dan 4 afbeeldingen toegevoegd worden
   migrations:
     acct: gebruikersnaam@domein van het nieuwe account
-    currently_redirecting: 'Jouw profiel wordt nu doorverwezen naar:'
+    currently_redirecting: "Jouw profiel wordt nu doorverwezen naar:"
```
Comment (SerCom-KC) [inline config/locales/nl.yml]: Please change it back to `'`

File: config/locales/nl.yml [hunk 23]
```diff
@@ -250,7 +252,7 @@ nl:
       nsfw:
         'false': Media tonen
         'true': Media verbergen
-      report: 'Gerapporteerde toot #%{id}'
+      report: "Gerapporteerde toot #%{id}"
```
Comment (SerCom-KC) [inline config/locales/nl.yml]: Please change it back to `'`

File: config/locales/nl.yml [hunk 24]
```diff
@@ -183,6 +184,7 @@ nl:
       update_failed_msg: Deze emoji kon niet worden bijgewerkt
       updated_msg: Bijwerken van emoji is geslaagd!
       upload: Uploaden
+      by_domain: Domein
```
Comment (SerCom-KC) [inline config/locales/nl.yml]: Please move this line below [line 163](https://github.com/jeroenpraat/mastodon/blob/cb5527e73002c60fda85514052a0e2c2d9ed68cc/config/locales/nl.yml#L163)

File: config/locales/nl.yml [hunk 25]
```diff
@@ -117,6 +117,7 @@ nl:
         admin: Beheerder
         moderator: Moderator
         user: Gebruiker
```
Comment (SerCom-KC) [inline config/locales/nl.yml]: Please move this line below [line 120](https://github.com/jeroenpraat/mastodon/blob/cb5527e73002c60fda85514052a0e2c2d9ed68cc/config/locales/nl.yml#L120)

Comment (SerCom-KC) [inline config/locales/nl.yml]: Please move this line below [line 120](https://github.com/jeroenpraat/mastodon/blob/cb5527e73002c60fda85514052a0e2c2d9ed68cc/config/locales/nl.yml#L120)

File: config/locales/nl.yml (hunk 24 above)
Comment (SerCom-KC) [inline config/locales/nl.yml]: Please move this line below [line 163](https://github.com/jeroenpraat/mastodon/blob/cb5527e73002c60fda85514052a0e2c2d9ed68cc/config/locales/nl.yml#L163)

File: config/locales/nl.yml (hunk 23 above)
Comment (SerCom-KC) [inline config/locales/nl.yml]: Please change it back to `'`

File: config/locales/nl.yml (hunk 22 above)
Comment (SerCom-KC) [inline config/locales/nl.yml]: Please change it back to `'`

File: config/locales/nl.yml (hunk 21 above)
Comment (SerCom-KC) [inline config/locales/nl.yml]: Please change it back to `'`

File: config/locales/nl.yml (hunk 20 above)
Comment (SerCom-KC) [inline config/locales/nl.yml]: Please change `🐘` back to `\U0001F418` and add `"` back

File: config/locales/nl.yml (hunk 18 above)
Comment (SerCom-KC) [inline config/locales/nl.yml]: Please change it back to `'`

File: config/locales/nl.yml (hunk 17 above)
Comment (SerCom-KC) [inline config/locales/nl.yml]: Please change it back to `'`

File: config/locales/nl.yml (hunk 16 above)
Comment (SerCom-KC) [inline config/locales/nl.yml]: Please change it back to `'`

File: config/locales/nl.yml (hunk 15 above)
Comment (SerCom-KC) [inline config/locales/nl.yml]: Please change it back to `'`

File: config/locales/nl.yml (hunk 14 above)
Comment (SerCom-KC) [inline config/locales/nl.yml]: Please change it back to `'`

File: config/locales/nl.yml (hunk 13 above)
Comment (SerCom-KC) [inline config/locales/nl.yml]: Please move this line below [line 603](https://github.com/jeroenpraat/mastodon/blob/cb5527e73002c60fda85514052a0e2c2d9ed68cc/config/locales/nl.yml#L603)

File: config/locales/nl.yml (hunk 12 above)
Comment (SerCom-KC) [inline config/locales/nl.yml]: Please add `"` back

File: config/locales/nl.yml (hunk 11 above)
Comment (SerCom-KC) [inline config/locales/nl.yml]: Please change it back to `'`

**Review by ghost — CHANGES_REQUESTED** [review]
Also do **NOT** change anything else.

**Comment by jeroenpraat** [comment]
Sorry for all the troubles. I downloaded the source files manually from Weblate and tried to fix a few things. First not enough and than I tried to fix more than needed (although I don't see any logic in using single or double quotes).

What did I learn: never ever manually use files from Weblate. The next time I want to do a manual update, I will make a diff locally and fix the existing Github files by hand.

**Review by ghost — APPROVED** [review]
`bundle exec i18n-tasks health nl` LGTM

**Comment by SerCom-KC** [comment]
Also about `app/javascript/mastodon/locales/nl.json`:
```
Untranslated keys:
  account.media: Media
  compose_form.publish: Toot
  compose_form.publish_loud: {publish}!
  confirmations.delete_list.confirm: Delete
  getting_started.appsshort: Apps
  getting_started.faq: FAQ
  keyboard_shortcuts.enter: to open status
  notifications.column_settings.reblog: Boosts:
  privacy.direct.short: Direct
  relative_time.days: {number}d
  relative_time.hours: {number}h
  relative_time.minutes: {number}m
  relative_time.seconds: {number}s
  search_popout.tips.hashtag: hashtag
  status.embed: Embed
  status.reblog: Boost
```

**Review by Gargron — APPROVED** [review]

**Comment by jeroenpraat** [comment]
@SerCom-KC The files are from [Weblate](https://weblate.joinmastodon.org), so those strings are in upstream issue. I don't have a development environment. I thought that wasn't needed for a simple translation.

**Pull request merged**